<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CameraAnimationDialog</class>
 <widget class="QWidget" name="CameraAnimationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>464</width>
    <height>478</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="3" column="2">
    <widget class="QPushButton" name="pushButton_8">
     <property name="text">
      <string>Load</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Player</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="2">
       <widget class="QLabel" name="labelTime">
        <property name="text">
         <string>Time: 0.00s</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="playPause">
        <property name="text">
         <string>Play</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="4">
       <widget class="QSlider" name="horizontalSlider">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>Stop</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="pushButton_6">
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QTreeWidget" name="treeWidget">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
     <property name="itemsExpandable">
      <bool>false</bool>
     </property>
     <property name="animated">
      <bool>true</bool>
     </property>
     <property name="expandsOnDoubleClick">
      <bool>false</bool>
     </property>
     <attribute name="headerVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Keyframe</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
   </item>
   <item row="7" column="2">
    <widget class="QPushButton" name="pushButton_3">
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="pushButton_4">
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QGroupBox" name="keyframeGroup">
     <property name="title">
      <string>Keyframe</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="time"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Time</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="pushButton_5">
        <property name="text">
         <string>Set</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="pushButton_7">
        <property name="text">
         <string>Go to</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>CameraAnimationDialog</receiver>
   <slot>onKeyframeAdd()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>137</x>
     <y>264</y>
    </hint>
    <hint type="destinationlabel">
     <x>463</x>
     <y>471</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_5</sender>
   <signal>clicked()</signal>
   <receiver>CameraAnimationDialog</receiver>
   <slot>onKeyframeSet()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>81</x>
     <y>419</y>
    </hint>
    <hint type="destinationlabel">
     <x>320</x>
     <y>477</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>time</sender>
   <signal>valueChanged(double)</signal>
   <receiver>CameraAnimationDialog</receiver>
   <slot>onKeyframeTimeSet(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>212</x>
     <y>446</y>
    </hint>
    <hint type="destinationlabel">
     <x>334</x>
     <y>477</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>treeWidget</sender>
   <signal>activated(QModelIndex)</signal>
   <receiver>CameraAnimationDialog</receiver>
   <slot>onKeyframeSelected()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>104</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel">
     <x>224</x>
     <y>1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>playPause</sender>
   <signal>clicked()</signal>
   <receiver>CameraAnimationDialog</receiver>
   <slot>onPlay()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>120</x>
     <y>362</y>
    </hint>
    <hint type="destinationlabel">
     <x>463</x>
     <y>397</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_6</sender>
   <signal>clicked()</signal>
   <receiver>CameraAnimationDialog</receiver>
   <slot>onAnimationSave()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>303</x>
     <y>264</y>
    </hint>
    <hint type="destinationlabel">
     <x>463</x>
     <y>405</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_7</sender>
   <signal>clicked()</signal>
   <receiver>CameraAnimationDialog</receiver>
   <slot>onKeyframeGoTo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>191</x>
     <y>420</y>
    </hint>
    <hint type="destinationlabel">
     <x>282</x>
     <y>477</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_8</sender>
   <signal>clicked()</signal>
   <receiver>CameraAnimationDialog</receiver>
   <slot>onAnimationLoad()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>454</x>
     <y>264</y>
    </hint>
    <hint type="destinationlabel">
     <x>328</x>
     <y>383</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>CameraAnimationDialog</receiver>
   <slot>onClear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>439</x>
     <y>434</y>
    </hint>
    <hint type="destinationlabel">
     <x>464</x>
     <y>302</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>CameraAnimationDialog</receiver>
   <slot>onAnimationSeek(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>194</x>
     <y>328</y>
    </hint>
    <hint type="destinationlabel">
     <x>200</x>
     <y>295</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>CameraAnimationDialog</receiver>
   <slot>onStop()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>158</x>
     <y>361</y>
    </hint>
    <hint type="destinationlabel">
     <x>5</x>
     <y>385</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>CameraAnimationDialog</receiver>
   <slot>onKeyframeDelete()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>84</x>
     <y>280</y>
    </hint>
    <hint type="destinationlabel">
     <x>216</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onKeyframeAdd()</slot>
  <slot>onKeyframeSet()</slot>
  <slot>onKeyframeTimeSet(double)</slot>
  <slot>onKeyframeSelected()</slot>
  <slot>onPlay()</slot>
  <slot>onAnimationSave()</slot>
  <slot>onKeyframeGoTo()</slot>
  <slot>onAnimationLoad()</slot>
  <slot>onClear()</slot>
  <slot>onAnimationSeek(int)</slot>
  <slot>onStop()</slot>
  <slot>onKeyframeDelete()</slot>
 </slots>
</ui>
